File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.java
class Solution {
    public boolean check(int[] nums) {
        int i;
        int c=0;
        int l=(nums.length)-1;
        if(nums.length<=2)
        {
             return true;
        }
       
        else{
        for(i=1;i<nums.length;i++)
        {
           if(nums[i]>=nums[i-1])
           {
            continue;
           }
           else{
            c++;
           }
        }
        
        if(c==1){
            if(nums[l]<=nums[0])
            {
                return true;
            }
            else{
        return false;
            }
        
    }
    else if(c==0)
    {
        return true;
    }
    return false;
    }
}
}

File: 1878-check-if-array-is-sorted-and-rotated/check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        x=0
        for i in range(1,len(nums)):
            if(nums[i-1]>nums[i]):
                x=i

        arr=sorted(nums)
        for i in range(len(nums)):
            if(arr[i] != nums[(i+x)%len(nums)]):
                return False
        return True

        

            

            
        